### ISSUE TITLE
BUG: filters do not work if no generic one is defined

### ISSUE THREAD

**Issue by stefanpante** [digipolisantwerp/smart-table_widget_angular#51]
## Steps to reproduce:

Change the display of the smartfilter from generic to optional

line 85 in projects/example-api/api.js

change this 
```javascript
....
    filters: [{
      "id": "smartfilter",
      "display": "generic",
      "type": "input",
      "label": "Zoek een film",
      "placeholder": "Zoek op titel, jaar, ...",
      "fields": ["movie_title", "director_name", "title_year"],
    },
...
```
to this

```javascript
....
    filters: [{
      "id": "smartfilter",
      "display": "optional",
      "type": "input",
      "label": "Zoek een film",
      "placeholder": "Zoek op titel, jaar, ...",
      "fields": ["movie_title", "director_name", "title_year"],
    },
...
```
## results
all filters stop working, no API calls are made.

**Comment by stefanpante** [comment]
I think that this has been an issue for the past few versions, 
last version for us without known issues is 1.5.7.

**Comment by MichaelCastiau** [comment]
https://github.com/digipolisantwerp/smart-table_widget_angular/pull/52

**Comment by MichaelCastiau** [comment]
@stefanpante Fixed in version 1.9.1
